src/FluentlyHttpClient/Utils/GqlQuery.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ReSharper disable once CheckNamespace
+namespace FluentlyHttpClient
```

Comment:
Can we move all these GQL related stuff to a sub folder named `GraphQL` and create
 - `FluentHttpClientGqlExtensions` (without namespace like this)
 - `GqlQuery`
 - `GqlResponse`